pkg/auth/auth.go

```diff
@@ -239,24 +270,25 @@ func Logout(systemContext *types.SystemContext, opts *LogoutOptions, args []stri
 		return nil
 	}
 
+	serverDomain := domain(server)
 	err = config.RemoveAuthentication(systemContext, server)
 	switch errors.Cause(err) {
 	case nil:
 		fmt.Fprintf(opts.Stdout, "Removed login credentials for %s\n", server)
 		return nil
 	case config.ErrNotLoggedIn:
-		authConfig, err := config.GetCredentials(systemContext, server)
+		authConfig, err := getCredentials(systemContext, args[0])
```

Comment:
1. `len(args)` might be 0
2. At least conceptually it’s cleaner to separate the part that extracts parameters from the CLI and the part that works with the data. Long-distance use of `args[0]` like this are more difficult to follow.

Reply:
Fixed